src/components/desktop/desktop-next-query-preview.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <NextQueryPreview
+    #default="{ results, totalResults, suggestion }"
+    class="x-list x-background--neutral-95 x-list--gap-06 x-padding--06"
+    :suggestion="nextQuery"
+  >
+    <span class="x-title2 x-font-weight--bold">{{ $t('nextQueryPreview.title') }}</span>
+    <div class="x-list x-list--horizontal x-list--align-center x-list--gap-03">
+      <span class="x-title3 x-font-weight--bold">
+        {{
+          $t('nextQueryPreview.query', {
+            query: suggestion.query
+          })
+        }}
+      </span>
+      <span class="x-text">
+        {{
+          $t('nextQueryPreview.totalResults', {
+            totalResults: totalResults
+          })
+        }}
+      </span>
+    </div>
+    <div class="x-list x-list--horizontal x-list--justify-space-between x-list--gap-05">
```

Comment:
> The description of the task said that a sliding panel should be rendered, however I can't see it. It is complex to style results width if we add it though. Can we confirm that we don't want an SlidingPanel here?

I've talked with @Alicia about the designs of this component. The `Sliding Panel` is only used in the `mobile` version, rendering an unlimited number of items. The `desktop` version, however, does not render a `Sliding Panel` and is limited to a maximum of 4 items.

Reply:
> Because of the justify space between, when there are a few items it spaces them:

This issue should be resolved after the refactor using a `BaseGrid`.